tfjs-backend-webgpu/package.json

```diff
@@ -11,13 +11,13 @@
   "scripts": {
     "benchmark": "yarn test --//:grep=benchmark",
     "build-ci": "yarn build",
-    "build": "bazel build :tfjs-backend-webgpu_pkg",
-    "bundle": "bazel build :tfjs-backend-webgpu_pkg",
+    "build": "yarn --cwd .. bazel build tfjs-backend-webgpu:tfjs-backend-webgpu_pkg",
+    "bundle": "yarn build",
     "bundle-ci": "yarn bundle",
-    "build-npm": "bazel build :tfjs-backend-webgpu_pkg",
-    "publish-npm": "bazel run :tfjs-backend-webgpu_pkg.publish",
-    "test": "yarn test-dev",
-    "test-dev": "ibazel run :tfjs-backend-webgpu_test --test-output=streamed"
+    "build-npm": "yarn build",
+    "publish-npm": "yarn --cwd .. bazel run tfjs-backend-webgpu:tfjs-backend-webgpu_pkg.publish",
+    "test": "yarn --cwd .. bazel test tfjs-backend-webgpu:tfjs-backend-webgpu_test --test_output=streamed",
+    "test-dev": "yarn --cwd .. bazel run tfjs-backend-webgpu:tfjs-backend-webgpu_test --test_output=streamed"
```

Comment:
Why to use `bazel test` in L19, but `bazel run` in L20? @haoyunfeix @qjia7

Reply:
I had the same question yesterday, and Yunfei answered me as below (https://www.npmjs.com/package/@bazel/concatjs):
When executed under bazel test, this uses a headless browser for speed. This is also because bazel test allows multiple targets to be tested together, and we don't want to open a Chrome window on your machine for each one. Also, under bazel test the test will execute and immediately terminate.
Running under ibazel test gives you a "watch mode" for your tests. The rule is optimized for this case - the test runner server will stay running and just re-serve the up-to-date JavaScript source bundle.
To debug a single test target, run it with bazel run instead. This will open a browser window on your computer. Also you can use any other browser by opening the URL printed when the test starts up. The test will remain running until you cancel the bazel run command.

So bazel run is more interactive than bazel test, per my understanding.

Reply:
Now I'm seeing this error on Linux during `yarn test`:

```
29 04 2022 05:33:46.922:ERROR [launcher]: No binary for Chrome browser on your platform.
  Please, set "CHROME_BIN" env variable.
```

But I've exported `CHROME_BIN`. Using `bazel run` seems to work well for me.

Reply:
Do you know if the other backends (WASM / WebGL) will use the system browser or `CHROME_BIN` during testing?

Reply:
The Karma Chrome launcher seems to have built-in support for Chrome Canary, so I'm going to see if that works and avoid abusing the `CHROME_BIN` environment variable.